spec/features/hiring_staff_can_filter_vacancies_in_their_dashboard_spec.rb

```diff
@@ -48,7 +48,7 @@
       scenario 'it shows all published vacancies' do
         visit jobs_with_type_organisation_path(:published)
 
-        expect(page).to have_content('Showing all jobs')
```

Comment:
Are we definitely getting rid of this text?

Reply:
again it was removed as it is part of an old figma design, i checked with chris